base_rest/http.py

```diff
@@ -202,11 +203,17 @@ def make_json_response(self, data, headers=None, cookies=None):
 
 def get_request(self, httprequest):
     db = httprequest.session.db
```

Comment:
@nilshamerlinck We should use the method `db_monodb` https://github.com/odoo/odoo/blob/13.0/odoo/http.py#L1493. to get the DB. Thanks to this method, if the db is not yet into the session we'll get the default one if only one DB is available for Odoo. What do you think about?

Reply:
Before calling `get_request()`:
- odoo calls `setup_db()` [here](https://github.com/odoo/odoo/blob/13.0/odoo/http.py#L1421)
- that already calls `db_monodb()` [here](https://github.com/odoo/odoo/blob/13.0/odoo/http.py#L1344) if session has no legit db

The `db:` condition is here to deal with all the other cases where odoo can't automatically choose a db (because there are none or many), but we still want `base_rest` to be loaded as `server_wide_modules` to address the "first request" issue on fresh workers.